deploy/gk-deploy

```diff
@@ -239,7 +239,7 @@ while [[ $# -ge 1 ]]; do
   shift
 done
 
-if [[ ! -f ${TOPOLOGY} ]]; then
+if [[ ${ABORT} -ne 1 ]] && [[ ! -f ${TOPOLOGY} ]]; then
```

Comment:
@jarrpa I agree, so Will grep the labels in #81 .  Abort is a risky operation and user should know it is gonna remove all gluster when used.